lib/mock-server/pretender-config.js

```diff
@@ -47,11 +48,65 @@ const defaultPassthroughs = [
   },
 ];
 
+const defaultRouteOptions = {
+  coalesce: false,
+  timing: undefined,
+};
+
+/**
+ * Determine if the object contains a valid option.
+ *
+ * @method isOption
+ * @param {Object} option An object with one option value pair.
```

Comment:
>An object with one option value pair

Does this mean that only one option can be set at a time?  From reading the code, it looks like it's checking that there's _at least_ one option key being used, but there could be other keys as well.

Reply:
It was the code that was already there supporting this, I just moved it into the extracted version of the pretender code.